README.md

```diff
@@ -147,15 +147,15 @@ Update [values.yaml](https://github.com/apiclarity/apiclarity/blob/master/charts
    ```
 
 3. Run backend and frontend locally using demo data:
-
-   ```shell
-   DATABASE_DRIVER=LOCAL FAKE_TRACES=true FAKE_TRACES_PATH=./backend/pkg/test/trace_files \
-   ENABLE_DB_INFO_LOGS=true ./backend/bin/backend run
-   ```
    Note: You might need to delete the local state file, and local db:
```

Comment:
Note: You might need to delete the **old** local state file and local db:

Reply:
maybe better to add and emphasize OLD to make it more clear when/why the user should do it